BadHabbit/BadHabbit/Kit/DesignKit/Models/LayoutConstants.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  LayoutConstants.swift
+//  BadHabbit
+//
+//  Created by Viktoria Misiulia on 25/07/2023.
+//
+
+import Foundation
+
+enum LayoutConstants {
+    enum ActionButton {
+        static let height = 50
+        static let leadingInset = 16
+        static let trailingInset = 16
```

Comment:
leading and trailing are common not only for ActionButton, but also for labels when leading and trailing constraints equals to superview, so it's LayoutConstants constants

Reply:
leadingInset and trailingInset was removed from this enum